devimint/src/lib.rs

```diff
@@ -201,6 +201,7 @@ pub async fn dev_fed(process_mgr: &ProcessManager) -> Result<DevFed> {
     cmd!(fed, "join-federation", fed.invite_code()?)
         .run()
         .await?;
+    info!(LOG_DEVIMINT, "await gateways registered");
```

Comment:
unrelated, but ok

Reply:
I have no idea how did this got here really... . It doesn't hurt, but ... weird.